frontend/src/features/Login/LoginForm.tsx

```diff
@@ -1,50 +1,68 @@
-import React, { ReactElement, useState } from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
 import { Button, Spin, Form } from 'antd';
-
 import * as dsnpLink from '../../dsnpLink';
-import { UserAccount } from '../../types';
+import { UserAccount, Handle } from '../../types';
 import { getContext } from '../../service/AuthService';
 import { getLoginOrRegistrationPayload, setConfig } from '@projectlibertylabs/siwf';
 
-/**
- * Props for the LoginForm component.
- */
 interface LoginProps {
-  /**
-   * Callback function to be called when the user logs in.
-   * @param account - The user account information.
-   */
   onLogin: (account: UserAccount) => void;
-
-  /**
-   * The ID of the provider used for authentication.
-   */
   providerId: string;
-
-  /**
-   * The URL of the Frequency RPC endpoint.
-   */
   nodeUrl: string;
-
-  /**
-   * The URL of the SIWF/Wallet-Proxy server.
-   */
   siwfUrl: string;
 }
 
-/**
- * Represents the Login component.
- *
- * @component
- * @param {Object} props - The component props.
- * @param {Function} props.onLogin - The callback function to be called when the user logs in.
- * @param {string} props.providerId - The ID of the provider.
- * @param {string} props.nodeUrl - The URL of the Frequency RPC endpoint.
- * @param {string} props.siwfUrl - The URL where Wallet-Proxy lives.
- * @returns {JSX.Element} The rendered Login component.
- */
+interface AccountData {
+  handle?: Handle;
+  expires?: number;
+  accessToken?: string;
+  msaId?: string;
+}
+
 const LoginForm = ({ onLogin, providerId, nodeUrl, siwfUrl }: LoginProps): ReactElement => {
   const [isLoading, setIsLoading] = useState(false);
+  const [accountData, setAccountData] = useState<AccountData>({});
+
+  useEffect(() => {
+    const sse = new EventSource(`${process.env.REACT_APP_BACKEND_URL}/auth/events`);
+
+    const handleAccountCreated = (e: MessageEvent) => {
+      const data = JSON.parse(e.data);
+
+      if (!data.handle || !data.msaId) {
+        return;
+      }
+
+      setAccountData((prev) => {
+        const newData = {
+          ...prev,
+          handle: data.handle,
+          msaId: data.msaId,
+        };
+        return newData;
+      });
+    };
```

Comment:
How do we know this is the account creation this client is waiting for? The backend is broadcasting ALL 'account_created' events.